src/stores/settings.js

```diff
@@ -33,7 +38,8 @@ export const useSettingsStore = defineStore('settings', {
 	state: () => ({
 		readStatusPrivacy: loadState('spreed', 'read_status_privacy', PRIVACY.PRIVATE),
 		typingStatusPrivacy: loadState('spreed', 'typing_privacy', PRIVACY.PRIVATE),
-		showMediaSettings: {}
+		showMediaSettings: {},
+		mediaDefaultState: getTalkConfig('local', 'call', 'start-without-media'),
```

Comment:
```suggestion
		startWithoutMedia: getTalkConfig('local', 'call', 'start-without-media'),
```

I was confused with the name `mediaDefaultStateEnabled`, when we disable everything on true.
Maybe change it to something close to API given?

Reply:
I am always open to any renaming :p